src/enzo/select_fft.F90

```diff
@@ -270,39 +270,39 @@ subroutine fortfft(x, rank, dim1, dim2, dim3, dir)
           select case ( choice )
 
             case default
-!             write(0,'("1D Stride 1 call to S90 FFT")')
+             write(0,'("1D Stride 1 call to S90 FFT")')
               call wrapper1d(x, rank, dim1, dim2, dim3, dir, s90_st1)
```

Comment:
Why are the comments to the write removed?

Reply:
fixed